apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -48,13 +51,68 @@ public final class MigratorApp {
   static final List<String> hibernateValidationModelPackages =
       List.of("gov.cms.bfd.model.rda", "gov.cms.bfd.model.rif");
 
+  /** This {@link ProcessHandle#pid} value is the migrator app's current process id (pid). */
+  private static final String PID = Long.toString(ProcessHandle.current().pid());
+
+  /** The file name of the {@link #PID} file to externally signal the migrator app's status. */
+  private static final String PID_FILENAME = String.format("%s.pid", PID);
+
+  /**
+   * This method is called to create a file that encodes the migrator app's process id as the file's
+   * name and stores the migrator app's status upon exit. This file should remain empty until the
+   * application has exited.
+   */
+  private static void createPidFile() {
+    try {
+      File pidFile = new File(PID_FILENAME);
+      if (pidFile.createNewFile()) {
+        LOGGER.info(String.format("Created new pid file %s", PID_FILENAME));
+      } else {
+        LOGGER.info(String.format("Found existing %s file", PID_FILENAME));
```

Comment:
Might be better as debug level; i dont see this piece of info being generally useful.

Reply:
I might agree, but this is only a single line for each instance of the application. I think the nature of this particular application might demand debugging on by default anyway, right?